diesel_cli/src/cli.rs

```diff
@@ -289,6 +302,12 @@ pub fn build_cli() -> Command {
                 .action(clap::ArgAction::Append)
                 .number_of_values(1)
                 .help("A list of derives to implement for every automatically generated SqlType in the schema, separated by commas."),
+        )
+        .arg(
+            Arg::new("print-schema-key")
+                .long("print-schema-key")
+                .action(ArgAction::Append)
+                .help("select schema key from diesel.toml, use 'default' for print_schema without key."),
```

Comment:
Can we default to the default schema key if that's not set. If that's already the case maybe indicate that in the help.